### ISSUE TITLE
Set up the project governance

### ISSUE THREAD

**Issue by maximlt** [holoviz-topics/examples#474]
Originally a sub-task of https://github.com/holoviz-topics/examples/issues/238.

Probably similarly to what was done for holoviz.org (https://github.com/holoviz/holoviz).

**Comment by jbednar** [comment]
Yes, I think the examples org should just follow the regular HoloViz org governance.
